trunk/ansible/install/run-install.py

```diff
@@ -263,7 +263,7 @@ def filepath_prompt(question):
     mpf_home = '/opt/mpf'
 
 # Get the package configuration file(s)
-package_config_files = sorted(glob.glob(''.join([mpf_home, '/manage/repo/files/mpf-*-package.json'])),
+package_config_files = sorted(glob.glob(''.join([mpf_home, '/manage/repo/files/openmpf-*-package.json'])),
```

Comment:
The other .json files in /home/mnt/mpfdata/jenkins/scripts/config_files will have to be renamed. You've renamed one of them so far.